pkg/blobstore/local/flat_blob_access.go

```diff
@@ -28,6 +29,26 @@ var (
 			Buckets:   append([]float64{0}, prometheus.ExponentialBuckets(1.0, 2.0, 16)...),
 		},
 		[]string{"storage_type", "operation"})
+
+	flatBlobAccessRefreshLatencySeconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "buildbarn",
+			Subsystem: "blobstore",
+			Name:      "flat_blob_access_refresh_latency_seconds",
```

Comment:
Here we use `_latency_` as a midfix, whereas the rest of our codebase uses `_duration_`. Could we please make this more consistent?

Also on the MetricsBlobAccess side, we use `_operations_` (plural). Probably good to stick to `_refreshes_` for consistency?